src/js/common/components/asyncLoader.jsx

```diff
@@ -0,0 +1,30 @@
+// Adapted from https://gist.github.com/acdlite/a68433004f9d6b4cbc83b5cc3990c194
+import React from 'react';
+import LoadingIndicator from '../../common/components/LoadingIndicator';
+
+export default function asyncLoader(getComponent, message) {
+  return class AsyncComponent extends React.Component {
+    static Component = null;
```

Comment:
Why static? I feel like this could come back to bite us later...If we're lazy-loading sibling or parent-child components, perhaps.

Then again, it doesn't look like the static variable ever gets changed, but if not, why not just set the initial
```js
state = { Component: null };
```
?